distribution/sender/routes/sender-activemq-consumer.xml

```diff
@@ -0,0 +1,93 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="sender-activemq-consumer" errorHandlerRef="shutdownErrorHandler">
+        <from uri="activemq:openmrs.sync.{{db-sync.senderId}}?connectionFactory=activeMqConnFactory&amp;acknowledgementModeName=CLIENT_ACKNOWLEDGE&amp;messageListenerContainerFactory=customMessageListenerContainerFactory&amp;asyncStartListener=true" />
+
+        <log message="Received message -> ${body}" />
+        
+        <setProperty name="requestUuid">
```

Comment:
requestUuid only exists for a sync messages when responding to a sync request, so you should only expect requestUuid to exist if operation.operation is set to **r** in the incoming message, so rather change line 16 below to use check for this rather than doing here to avoid using suppressExceptions